libraries/chain/include/eosio/chain/webassembly/interface.hpp

```diff
@@ -472,6 +472,26 @@ namespace webassembly {
          */
          bool is_account(account_name account) const;
 
+         /**
+          * Retrieves the code hash for an account, if any.
+          *
+          * @ingroup authorization
+          * @param account - name of the account to check.
+          * @param code_sequence - Receives the number of times code has changed, or 0 if account does not exist.
+          * @param code_hash - Receives the code_hash, or empty if code not set or account does not exist.
+          * @param vm_type - Receives the vm type, or 0 if code not set or account does not exist.
+          * @param vm_version - Receives the vm version, or 0 if code not set or account does not exist.
+          *
+          * @return true if the account exists and has code.
+          * @return false otherwise.
```

Comment:
Is the result bool necessary, given that it is strictly equivalent to `code_hash != sha256{}`?